### ISSUE TITLE
Standard LED color

### ISSUE THREAD

**Issue by roggenrohl-inge** [roggenrohl-inge/wordclock#5]
Standard LED color is currently set to white. The web server should allow to adjust the default color.

**Comment by guus46a** [comment]
How can i change color, its stil white

**Comment by roggenrohl-inge** [comment]
Hi, although the project is public I somehow did not expect that somebody other than me or my friends, who are working on the project, would download and test the code. However, comments, questions, and contributions are always welcome. 

Some comments in the code and also the webserver are in German. I will change it to English so that everybody can understand. We may also add some further documentation if more people are interested in the project.

As to your question:
Currently, the color of the LEDs is stored in HSV format in the EEPROM. The hue value sets the color of the LEDs. A value between 0 and 1 corresponds to 0 to 360 degrees in the chromatic circle (see http://colorizer.org). By selecting a different hue value you can modify the LED color. We planned to integrate it into the web server, but this is still an open issue. By the way: You can access the webserver via "wordclock.local" in your browser.

Feel free to ask further questions, but please be patient with us.

**Comment by guus46a** [comment]
I have searced whole github to find the right wordclock to modify in Dutch. By far this is the best clock i did find, and i know there is a lot of work to be done before it is ready. Thanks for the color explanations and i will wait for the web server integration. Hope that even rainbow and other integrations are possible.

**Comment by roggenrohl-inge** [comment]
Hi, I couldn't test the lastest commit yet, but I integrated color control and other stuff in the web server. You can access it via "wordclock.local" in your browser. I also added a brief feature list in the readme file. An LDR measurement is now used to adjust LED brightness. 

Please ask if you have any questions.

**Comment by guus46a** [comment]
Hello, i have used the code and i saw that the website is chanched a lot. I can change the colors but the ldr didn't work. I put ldr on 5v, 10k resistor on ground and both other sides to a0 so it should work i think. Thanks for the nice work.

**Comment by roggenrohl-inge** [comment]
Hi, thanks for the feedback. Do you mean the color of the clock and ambilight LEDs? At the moment both have the same color. Having different colors is no big deal. I can upload the code during the next days.

About the LDR: I made a mistake in the readme. I said it has to be connected to 5V but it's 3.3 V.

**Comment by guus46a** [comment]
Hello, the problem that i have is that when i put flashlight on ldr nothing happens. And the other thing is that i have 82 leds and no extra leds, and i can't turn them in ambilight. Thanks in advance.

**Comment by roggenrohl-inge** [comment]
I will test the LDR again tomorrow. Did you check whether your LDR value is saturated? The value of the analog input should always be between above 0 and below 1023. It does not matter whether you have ambilight LEDs or not, i.e. the ambilight controls do not affect or depend on other functionalities.

**Comment by roggenrohl-inge** [comment]
LDR is working for me. I uploaded some updates, not everything is thoroughly tested though.

**Comment by guus46a** [comment]
I noticed that the ldr is working but it can take at most 60 seconds to change, i didn't realize that. Wordclock.local isn't working for me but i can go to the ip adres of the esp. What do you mean with ambilight, i thought that it should automaticly change colors? I saw you did a big job in rewriting the whole ino. Thanks

**Comment by roggenrohl-inge** [comment]
Yes, the clock is updated every minute. When I did it more frequently, I had problems with the web server. Our word clock has extra LEDs in the back which constitute the ambilight. The color is set manually, only the brightness is controlled via the LDR. So it is not comparable to the ambilight of modern TVs. If you only have the clock LEDs then the ambilight features will simply do nothing. When I have more time I will update the description of the project and upload some pictures.

**Comment by guus46a** [comment]
Aha thanks for the explanations, i thought that it was possible to change words in rainbow effect. I have code written for dutch with mask for letters if you like in dxf format.
![klok](https://cloud.githubusercontent.com/assets/27052039/26520172/e1945004-42cd-11e7-8f94-81f627aff4a2.jpg)
.
